wouso/interface/cpanel/forms.py

```diff
@@ -346,4 +346,6 @@ def __init__(self, data=None, players=None):
         super(KarmaBonusForm, self).__init__(data)
         for p in players:
             self.fields['%s' % p.user.username] = forms.IntegerField(label=unicode(p), initial=0, min_value=0,
-                                                                     required=True, help_text=" ")
\ No newline at end of file
+                                                                     required=True, help_text=" ")
+class ProposedQuestionReviewForm(forms.Form):
+    text = forms.CharField(required=True, max_length=2000, widget=forms.Textarea)
```

Comment:
@vladeugeniu newline at the end of the file and two lines before class definition